src/snowflake/snowpark/files.py

```diff
@@ -24,7 +29,14 @@
 else:
     from collections.abc import Iterable
 
-_DEFER_IMPLEMENTATION_ERR_MSG = "SnowflakeFile currently only works in UDF and Stored Procedures. It doesn't work locally yet."
+_NON_LOCAL_PATH_ERR_MSG = "SnowflakeFile currently supports only relative paths and read apis in local testing mode."
+_WRITE_MODE_ERR_MSG = (
+    "SnowflakeFile currently doesn't support write APIs in local testing mode."
+)
+_DEFER_IMPLEMENTATION_ERR_MSG = "Not yet supported in UDF and Stored Procedures."
+READ_MODES = ["r", "rb"]
+WRITE_MODES = ["w", "wb"]
```

Comment:
these two will be part of public apis.
are we expect users to use the two constants?